pandas/core/groupby/groupby.py

```diff
@@ -2013,8 +2022,13 @@ def _transform(self, func, *args, engine=None, engine_kwargs=None, **kwargs):
             # result to the whole group. Compute func result
             # and deal with possible broadcasting below.
             # Temporarily set observed for dealing with categoricals.
-            with com.temp_setattr(self, "observed", True):
```

Comment:
did this `com.temp_setattr(self, "observed", True)` get subsumed into `self._idxmin_idxmax`?  (could also be why the observed keyword got added to _get_grouper answering my question there?)

Reply:
If I recall right, this can be moved to a precursor because it wasn't actually doing anything. We cache all the relevant objects this would impact, which have already been computed. I'll double check and move this out of here.